index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./css/style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <main>
+    <div class="background-image"><img src="./images/Illustration.png"/ alt="Vector image"></div>
+    <header class="header-container">
+      <div class="header-title-container"><h2 class="header-title">John Doe</h2></div>
+      <div class="menu-icon-container"><a class="button header-menu-icon" href="#"><img src="./images/Menu.svg" alt="Menu icon"/></a></div>
+    </header>
+    <section class="headline-container">
+      <h1 class="headline-title">Hey There.<br>I’m Alex</h1>
+      <h2 class="headline-subtitle">I’am a Software Developer</h2>
+      <p class="headline-text">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <div class="social-media-button-group">
+        <a class="button social-media-button" href="#"><img src="./images/GitHub.svg" alt="Github icon"/></a>
+        <a class="button social-media-button" href="#"><img src="./images/Linkedin.svg" alt="Linkedin icon"/></a>
+        <a class="button social-media-button" href="#"><img src="./images/Angelist.svg" alt="Angelist icon"/></a>
+        <a class="button social-media-button" href="#"><img src="./images/Twitter.svg" alt="Twitter icon"/></a>
+        <a class="button social-media-button" href="#"><img src="./images/Medium.svg" alt="Medium icon"/></a>
+      </div>
```

Comment:
Please consider wrapping the social media icons in a `ul` tag and each `a` tag should also be wrapped in a `li` tag, also remember to include a link to each of your social media pages. 
A link per icon will let you connect to your social networks and other users can use them to connect with you.